src/main/java/reposense/util/SystemUtil.java

```diff
@@ -11,4 +14,18 @@
     public static boolean isWindows() {
         return (System.getProperty("os.name").toLowerCase().indexOf("win") >= 0);
     }
+
+    /**
+     * Returns true if the given String is a valid URL.
+     */
+    public static boolean isValidUrl(String possibleUrl) {
+        boolean isValidUrl;
+        try {
+            new URL(possibleUrl);
+            isValidUrl = true;
```

Comment:
would just `return true` and `return false` be cleaner without creating a new variable?

Reply:
Yes, that's simpler. Done